streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -103,6 +103,11 @@ public JoinWindows after(long timeDifference) {
     }
 
     @Override
+    public long size() {
```

Comment:
No. We need the window size when creating a Window Store as when we forward a windowed value we need to do `new Windowed(key, new TimeWindow(start, windowSize))`